edward/util/random_variables.py

```diff
@@ -714,6 +715,93 @@ def get_variables(x, collection=None):
   return list(output)
 
 
+def is_independent(a, b, condition=None):
+  """Assess whether a is independent of b given the random variables in
+  condition.
+
+  Implemented using the Bayes-Ball algorithm[1].
+
+  Args
```

Comment:
Fields like `Args` and `Returns` require a colon at the end, c.f., other function docstrings in this file.